injectors/index.js

```diff
@@ -77,7 +84,19 @@ const VALID_PLATFORMS = [ 'stable', 'ptb', 'canary', 'dev', 'development' ];
       '\n'
     );
     console.log('Try again with elevated permissions.');
+  } else if (e.code === 'ENOENT') {
+    console.log(
+      process.argv[2] === 'inject'
+        ? BasicMessages.PLUG_FAILED
+        : BasicMessages.UNPLUG_FAILED,
+      '\n'
+    );
+    console.log(
+      'Replugged wasn\'t able to inject itself due to missing files.',
```

Comment:
Be more specific and more clear. Just say Discord <version> could not be found. Also, maybe make this yellow/bold.